server/svix-server/tests/e2e_endpoint.rs

```diff
@@ -649,6 +649,68 @@ async fn test_list() {
     .unwrap();
 }
 
+#[tokio::test]
+async fn test_endpoint_list_ordering() {
+    let (client, _jh) = start_svix_server().await;
+
+    let app_id = create_test_app(&client, "App1").await.unwrap().id;
+
+    for i in 0..12 {
```

Comment:
No biggie, but 12 sounds like a large number (so slower tests), 5-6 would probably suffice (it adds up, and our tests are already too slow)

Reply:
Reduced to 5.